js/components/dial.js

```diff
@@ -1494,29 +1496,33 @@ var DT_dial = (function () {
     } else {
       me.min = choose(me.block.min, -10);
```

Comment:
Can you set the default min value to 0 for a kWh device, and -2000 for a P1 meter?

Reply:
Done